<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="trainTrack01">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>110</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: indigo</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrem01">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trainTrack04">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>110</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trainTrack02">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>330</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trainTrack03">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>330</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trainTrack06">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>330</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trainTrack05">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>330</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trainTrack07">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>130</y>
      <width>330</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: violet</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
